L40-node-call-invocation-transformer.md

```diff
@@ -0,0 +1,56 @@
+gRPC Node Call Invocation Transformer API
+----
+* Author(s): murgatroid99, WeiranFang
+* Approver: wenbozhu
+* Status: Draft
+* Implemented in: Node
+* Last updated: 26-09-2018
+* Discussion at: https://groups.google.com/forum/#!topic/grpc-io/f3v4SBvj7L4
+
+## Abstract
+
+Add a client API for transforming some paramters and intermediate variables for each call made using that client.
+
+## Background
+
+Some advanced call interception use cases require functionality that is not provided by the existing `CallInterceptor` API. 
+
```

Comment:
maybe add "E.g. being able to use a channel pool for a given service"?

Reply:
I added an example.